omrmakefiles/rules.ztpf.mk

```diff
@@ -77,6 +77,9 @@ ifneq (,$(findstring executable,$(ARTIFACT_TYPE)))
 	GLOBAL_LDFLAGS+=$(DEFAULT_LIBS)
 endif
 
+ZTPF_ROOT+=/ztpf/commit
```

Comment:
This will add `/ztpf/commit` to the end of `ZTPF_ROOT`.  If someone overrides this option on the command line or in the environment you will end up concatenating this value to what the user set.  I think you wanted this `ZTPF_ROOT?=/ztpf/commit` and `PROJECT_ROOT?=/projects/jvmport/userfiles`